src/server/chromium/crPage.ts

```diff
@@ -646,7 +646,7 @@ class FrameSession {
     session.once('Runtime.executionContextCreated', async event => {
       worker._createExecutionContext(new CRExecutionContext(session, event.context));
     });
-    Promise.all([
+    void Promise.all([
```

Comment:
Can just drop Promise.all.

Reply:
Yes, but it would be even more verbose as it'd require `void` prefix for each statement.

Reply:
Removed Promise.all and made replaced the methods with synchronous version.